mobile/native/src/calculations/mod.rs

```diff
@@ -31,3 +32,13 @@ pub fn calculate_liquidation_price(price: f64, leverage: f64, direction: Directi
 
     liquidation_price
 }
+
+pub fn calculate_fees() -> u64 {
+    // TODO: this should probably come from a configuration.
+    let fee_rate = 2;
+    let total_fee = (dlc::channel::sub_channel::dlc_channel_and_split_fee(fee_rate)
+        + dlc::util::weight_to_fee(LN_GLUE_TX_WEIGHT, fee_rate)) as f64;
+
+    // the party that offers the trade will pay the extra sat in case the fees are uneven.
+    (total_fee / 2.0).ceil() as u64
```

Comment:
If I understand this correctly than this is the _worst case_ fee which only applies if we force close the channel.

Reply:
yes, so in the best-case (or regular) scenario, these fees will never be paid. So I guess it would make sense to depict them somehow differently.

It's basically an additional collateral that we have to lock, in order to securely resolve the dlc channel in case the counterparty is trying to cheat.

This could even help the trader to understand that this is self-custodial.